src/providers/ipa/ipa_init.c

```diff
@@ -611,6 +611,11 @@ static errno_t ipa_init_misc(struct be_ctx *be_ctx,
         return ret;
     }
 
+    /* We must ignore entries in the views search base
+     * (default: cn=views,cn=accounts,$BASEDN) */
+    sdap_id_ctx->opts->sdom->ignore_search_bases = \
```

Comment:
Hi,

I would suggest to call this `ignore_user_search_bases` in case we might have to add ignored bases for group or other objects some time as well.

bye,
Sumit

Reply:
Done